Jenkinsfile.crossbuild

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env/ groovy
+@Library('cliqz-shared-library@vagrant') _
+
+/*
+ TRIGGERING JOB
+
+    ```groovy
+node(CROSS_BUILD_NODE){
+    step([
+        $class: 'CopyArtifact',
+        projectName: TRIGGERING_JOB_NAME,
+        selector: [$class: 'SpecificBuildSelector', buildNumber: TRIGGERING_BUILD_NUMBER],
+        target: 'artifacts'
+    ])
+
+    def helpers = load "artifacts/build-helpers.groovy"
+    stage("Checkout") {
+        helpers.checkoutSCM(REPO_URL, COMMIT_ID)
+    }
+    load ENTRY_POINT
+}
+    ```
+*/
+def imageName="browser-f/crossbuild"
+def image
+def depsVersion = ""
+
+depsVersion = readFile('mozilla-release/browser/config/version_display.txt').trim()
+
+withEnv(["depsVersion=$depsVersion"]){
+    sh """#!/bin/bash -l
+        set -e
+        set -x
+        cd docker/
+        ./get_osxcross_deps.sh
+    """
+}
+image = stage('Build docker image') {
+    //Build docker image. 
+    docker.build(imageName, '--no-cache --build-arg user=`whoami` --build-arg UID=`id -u` --build-arg GID=`id -g` -f docker/Dockerfile.osxcross ./docker/')
+}
+image.inside("-v ${pwd}:/opt/browser-f -w /opt/browser-f"){
+    withEnv([
+        "CQZ_RELEASE_CHANNEL=$CQZ_RELEASE_CHANNEL",
+        "CQZ_BUILD_DE_LOCALIZATION=$CQZ_BUILD_DE_LOCALIZATION"]){
+        stage('Biuld browser') {
+            sh """#!/bin/bash -l
+                set -e
+                set -x
+                ./magic_build_and_package.sh --clobber
+            """
+        }
+    }
+    stage('Stash artifacts'){
+        stash includes: 'obj/dist/*de.mac.dmg', name: 'dedmg'
+        stash includes: 'obj/dist/*US.mac.dmg', name: 'usdmg'
```

Comment:
please use full prefix for EN, it is "en-US"

Reply:
Done